mnt/reform2-nitrogen8m/kernel/linux-5.7.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPackages, fetchurl, perl, buildLinux, modDirVersionArg ? null, ... } @ args:
+
+with stdenv.lib;
```

Comment:
```suggestion
with lib;
```

Reply:
Good catch, I copied this from and old Nixpkgs.